packages/react-components/lib/tasks/task-table.tsx

```diff
@@ -19,7 +27,11 @@ const useStyles = makeStyles((theme) => ({
     },
   },
   phasesCell: {
-    padding: `0 ${theme.spacing(1)}px`,
+    padding: `0 ${theme.spacing(1)}px ${theme.spacing(1)}px ${theme.spacing(1)}px`,
```

Comment:
material already has a separator between rows, can we instead add a margin below the phases row so that it is more prominent?

```tsx
  phasesRow: {
    marginBottom: theme.spacing(1),
  },
```

```tsx
        <TableCell className={classes.phasesCell} colSpan={5}>
          <TaskPhases className={classes.phasesRow} taskSummary={task}></TaskPhases>
        </TableCell>
```

Reply:
I think this change is already in place. I will restore it back to the original version since it should look better once the task phases are out.